db/store.go

```diff
@@ -178,6 +178,40 @@ func (s *EventStore) GetCfAuditEvents(filter RawEventFilter) ([]cfclient.Event,
 	return events, nil
 }
 
+func (s *EventStore) LastSeenEvent() (*time.Time, error) {
+	ctx, cancel := context.WithTimeout(s.ctx, DefaultQueryTimeout)
+	defer cancel()
+	tx, err := s.db.BeginTx(ctx, &sql.TxOptions{ReadOnly: true})
```

Comment:
Do we actually need a transaction here if we're just doing one select from one table?